server/tools/package.json

```diff
@@ -6,6 +6,7 @@
     "application-config": "^1.0.1",
     "cli-table": "^0.3.1",
     "netrc-parser": "^3.1.6",
+    "moment": "^2.24.0",
```

Comment:
Please don't use moment. Check the date is valid manually, and create the a `Date` object:

```
const d = new Date('2019-07-29')

// 2019-07-29T00:00:00.000Z
```

Reply:
Good idea. Done!